test/wrappers/ortools_multi_trips_test.rb

```diff
@@ -0,0 +1,120 @@
+# Copyright © Mapotempo, 2019
```

Comment:
```suggestion
# Copyright © Mapotempo, 2021
```

Reply:
oh, we are in 2021 already ? xD thanks !